digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -462,10 +462,11 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
 
   def getValidSurroundingLinks(linkIds: Set[Long], floating: RoadAddressLink): Map[Long, Option[RoadAddressLink]] = {
     val roadLinks = roadLinkService.getViiteCurrentAndHistoryRoadLinksFromVVH(linkIds)._1
+    val vvhRoadLinks = roadLinkService.getViiteCurrentAndHistoryRoadLinksFromVVH(linkIds)._2
```

Comment:
These should be fetched together, not as two consecutive VVH requests (takes double the time)